test/e2e/common/test.go

```diff
@@ -147,20 +161,51 @@ func AfterEachCheck() error {
 		Version:  "v1alpha1",
 		Resource: "mayastorvolumes",
 	}
-	msvs, _ := gTestEnv.DynamicClient.Resource(msvGVR).Namespace(NSMayastor).List(context.TODO(), metaV1.ListOptions{})
+	msvs, err := gTestEnv.DynamicClient.Resource(msvGVR).Namespace(NSMayastor).List(context.TODO(), metaV1.ListOptions{})
+	if err != nil {
+		errorMsg += fmt.Sprintf("%s %v", errorMsg, err)
+	}
 	if len(msvs.Items) != 0 {
-		errorMsg += " found leftover MayastorVolumes"
-		logf.Log.Info("AfterEachCheck: found leftover MayastorVolumes, test fails.")
+		errorMsg += " found MayastorVolumes"
 	}
 
 	// Check that Mayastor pods are healthy no restarts or fails.
-	err := CheckTestPodsHealth(NSMayastor)
+	err = CheckTestPodsHealth(NSMayastor)
 	if err != nil {
-		errorMsg = fmt.Sprintf("%s %v", errorMsg, err)
+		errorMsg += fmt.Sprintf("%s %v", errorMsg, err)
 	}
 
+	err = CheckAllPoolsAreOnline()
+	if err != nil {
+		errorMsg += fmt.Sprintf("%s %v", errorMsg, err)
+		logf.Log.Info("BeforeEachCheck: all pools are not online")
```

Comment:
nit: message should be "not all pools are online" ?

Reply:
Fixed